build/Microsoft.DotNet.Cli.Version.props

```diff
@@ -0,0 +1,18 @@
+<Project>
+    <PropertyGroup>
+      <VersionMajor>2</VersionMajor>
+      <VersionMinor>0</VersionMinor>
+      <VersionPatch>0</VersionPatch>
+      <ReleaseSuffix>alpha</ReleaseSuffix>
+      <Channel>master</Channel>
+      <BranchName>master</BranchName>
+
+      <CliVersionPrefix>$(VersionMajor).$(VersionMinor).$(VersionPatch)-$(ReleaseSuffix)</CliVersionPrefix>
+      <SimpleVersion>$(VersionMajor).$(VersionMinor).$(VersionPatch).$(CommitCount)</SimpleVersion>
+      <VersionSuffix>$(ReleaseSuffix)-$(CommitCount)</VersionSuffix>
+      <NugetVersion>$(VersionMajor).$(VersionMinor).$(VersionPatch)-$(VersionSuffix)</NugetVersion>
+      
+      <SdkVersion>$(NugetVersion)</SdkVersion>
+      <SdkNugetVersion>$(SdkVersion)</SdkNugetVersion>
```

Comment:
Is there a reason we have this property?  Why can't the places just use `$(NuGetVersion)`?  I can understand having a `NuGetVersion` and a `SdkVersion`, because our SdkVersion may not always be the NuGet version.  But having `SdkNuGetVersion` and `NuGetVersion` makes little sense to me.